docs/book/src/tasks/certs/experimental-ca-rotation.md

```diff
@@ -0,0 +1,103 @@
+## Experimental CA rotation procedure
+
+<aside class="note warning">
+
+<h1>Cluster API CA rotation support</h1>
+
+CA rotation is not supported (nor automated) by Cluster API, please use this procedure at your own risk!
+Rotating CA is a tough Kubernetes topic on its own, it's even harder when everything is wrapped around Cluster API automation and immutability concepts.
+
+An issue is currently tracking discussions around this feature request : [Automatic CA rotation in CAPI #7721](https://github.com/kubernetes-sigs/cluster-api/issues/7721). Reading the issue is strongly advised to really be sure of what you are going to do.
+
+The recommended strategy in CAPI is to destroy and recreate clusters ahead of the expirty date to avoid CA expiry.
+
+</aside>
+
+## CA manual rotation step-by-step guide
+
+### Rotating `cluster-ca`, `cluster-proxy` & `cluster-etcd` CA Secrets
+
+During this step we're going to use the names `CA1`, `CA2`, `pubCA` and `CAkey`. These are:
+
+* `CA1` = old CA that needs to be replaced
+* `CA2` = new CA that will replace the old one and signs all new kubernetes certificates
+* `pubCA1+2` = means that you should concatenate CA cert1 and CA cert2 (in this order)
+* `CAkey2` = this is the private key for the new CA
+
+<h1>PREFLIGHT CA ROTATION OPERATIONS</h1>
```

Comment:
I'm not sure those are actual pre-flight, because we are already doing actual operations; also, H1 leads to a weird title nesting. what about simply dropping this title (and the following) and keeping a simple list of tasks